src/cavendish_particle_tracks/_magnification_dialog.py

```diff
@@ -132,9 +134,7 @@ def _on_click_add_coords_b2(self) -> None:
     def _add_coords(self, fiducial: int) -> list[float]:
         """When 'Add' is selected, the selected point is added to the corresponding fiducial text box"""
 
-        selected_points = self.parent._get_selected_points(
-            "Points_Calibration"
-        )
+        selected_points = self.parent._get_selected_points("Magnification")
```

Comment:
🐛 ?

Reply:
Nope, the layer should be called "Magnification", we might have changed the name since last time....